config/schedule.rb

```diff
@@ -60,3 +60,9 @@ def local(time)
 every 30.minutes, roles: [:cron] do
   runner "RegionsIntegrityCheck.sweep"
 end
+
+every 1.month, at: local("4:00 am"), roles: [:cron]  do
```

Comment:
I find it strange that we don't specify a day of the month, but we do the same above on the exotel job.

Reply:
Yep I confirmed it writes this:

```
0 4 1 * * /bin/bash -l -c 'cd /Users/tim/src/simple-server && RAILS_ENV=production bundle exec rake dhis2:export --silent >> /home/deploy/apps/simple-server/shared/log/cron.log 2>&1'
```